src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/GCRefMapBuilder.cs

```diff
@@ -56,14 +56,14 @@ public class GCRefMapBuilder
         /// </summary>
         private readonly TransitionBlock _transitionBlock;
 
-        public GCRefMapBuilder(NodeFactory factory, bool relocsOnly)
+        public GCRefMapBuilder(TargetDetails target, bool relocsOnly)
```

Comment:
I like this, it didn't occur to me we could do with just TargetDetails in the builder. Thank you for cleaning this up!